app/uk/gov/hmrc/individualsemploymentsapi/domain/integrationframework/IfEmploymentEntry.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsemploymentsapi.domain.integrationframework
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
+import play.api.libs.json._
+
+case class IfEmploymentEntry(id: IfId, employments: Seq[IfEmployment])
+
+object IfEmploymentEntry {
+  val createEmploymentEntryFormat: Format[IfEmploymentEntry] = Format(
+    (
+      (JsPath \ "id").read[IfId] and
```

Comment:
I do not think we will have an ID in the response JSON. 

I think this is a hangup from dynamic stub. I think we need to drop this from this object.